azure-communication-ui/azure-communication-ui/src/calling/java/com/azure/android/communication/ui/calling/service/sdk/CallingSDKWrapper.kt

```diff
@@ -291,6 +332,16 @@ internal class CallingSDKWrapper(
         return result
     }
 
+    // predefined order to return camera
+    private fun getCameraByFacingTypeSelection(): VideoDeviceInfo? {
+        return getCamera(CameraFacing.FRONT) ?: getCamera(CameraFacing.BACK)
```

Comment:
functionally could be nicer code. Chain of elvis operators isn't the nicest looking solution.

I.e.
listOf(CameraFacing.FRONT, CameraFacing.BACK, CameraFacing.External).forEach {
  val camera = getCamera(it)
  if (camera != null) {
   return camera
  } 
}